### ISSUE TITLE
concept relations and modification of concept labels for "sibling"

### ISSUE THREAD

**Issue by LinguList** [clld/concepticon-data#402]
```
3037	SAME-SEX OLDER SIBLING	Kinship	A person of the same sex who is born before another person who has the same parents.	Person/Thing	
3038	SAME-SEX YOUNGER SIBLING	Kinship	A person of the same sex who is born after another person who has the same parents.	Person/Thing	
3039	DIFFERENT-SEX OLDER SIBLING	Kinship	A person of the different sex who is born before another person who has the same parents.	Person/Thing	
3040	DIFFERENT-SEX YOUNGER SIBLING	Kinship	A person of the different sex who is born after another person who has the same parents.	Person/Thing	

```

These concept sets need to be refined. First, we need concept relations to already existing SIBLING terms in concepticon, second, the SAME-SEX would be ideally put into brackets.

**Comment by SimonGreenhill** [comment]
This is something that might be helped by Kinbank (@Supertyp)?

**Comment by LinguList** [comment]
Yes, they are working on it, but they tend to split all types, while concepticon has all kinds of degrees (coarse, narrow). I won't really think of changing too much of the concepticon_gloss here soon, as once the vanikin/parabank list is finalized, we will take over their proposed labels.

But I think, at some point, we need a policy for broader/narrower relations. I prefer to only show the closest neighbor, and this could also be automatically calculated, I guess. In any case, with the recently submitted list on Sun-1991-1004 and other work by @tresoldi, we will have even more concepts with kinship terms than before. Having vanikin/parabank finalised will therefore be very useful for us.

**Comment by AnnikaTjuka** [comment]
Should we change the concept labels as follows or should we wait until the vanikin/parabank list (#391) is added?

ID | concept gloss | new suggestion
-- | -- | --
3037 | SAME-SEX OLDER SIBLING | OLDER SIBLING (SAME-SEX)
3038 | SAME-SEX YOUNGER SIBLING | YOUNGER SIBLING (SAME-SEX)
3039 | DIFFERENT-SEX OLDER SIBLING | OLDER SIBLING (DIFFERENT-SEX)
3040 | DIFFERENT-SEX YOUNGER SIBLING | YOUNGER SIBLING (DIFFERENT-SEX)

**Comment by SimonGreenhill** [comment]
Kinbank is getting close - here's the [list of concepts](https://github.com/SimonGreenhill/kinbank/blob/master/kinbank/cldf/parameters.csv), but this will be a lot of work.

**Comment by AnnikaTjuka** [comment]
> Kinbank is getting close - here's the [list of concepts](https://github.com/SimonGreenhill/kinbank/blob/master/kinbank/cldf/parameters.csv), but this will be a lot of work.

Fascinating! And cool that you have the Daakaka and Daakie systems in there!
